debian/control

```diff
@@ -0,0 +1,16 @@
+Source: imx-firmware
+Section: contrib/electronics
+Priority: optional
+Build-Depends: debhelper-compat (= 12)
+Standards-Version: 4.3.0
```

Comment:
I got 3 warnings:
W: imx-firmware source: changelog-should-mention-nmu
W: imx-firmware source: source-nmu-has-incorrect-version-number 5.10.72-2.2.2-1
W: firmware-nxp: new-package-should-close-itp-bug

The first and second warning are "nmu"...
I though that I can ignore the last one -> [http://lintian.debathena.org/tags/new-package-should-close-itp-bug.html](url)

Reply:
All 3 are just related to some debian workflows. We can ignore them.
But you said that you get warnings if you use a newer standards-version

Reply:
> All 3 are just related to some debian workflows. We can ignore them. But you said that you get warnings if you use a newer standards-version

Sorry...
As you said, I updated the version to 4.6.1 -> [Please see here](https://www.debian.org/doc/debian-policy/upgrading-checklist.html)
Whit that version I got this warning:
`W: imx-firmware source: newer-standards-version 4.6.1 (current is 4.3.0)`

That is the reason because I used the Version 4.3.0...